src/tss2-tcti/tcti-device.c

```diff
@@ -118,7 +119,7 @@ tcti_device_receive (
     TSS2_TCTI_DEVICE_CONTEXT *tcti_dev = tcti_device_context_cast (tctiContext);
     TSS2_TCTI_COMMON_CONTEXT *tcti_common = tcti_device_down_cast (tcti_dev);
     TSS2_RC rc = TSS2_RC_SUCCESS;
-    ssize_t  size;
+    ssize_t size = 0;
```

Comment:
This good cleanup but it doesn't belong in this commit.

Reply:
The size needs to be initialized because the compiler complains otherwise